app/src/main/java/at/roteskreuz/stopcorona/di/ContextDependentModule.kt

```diff
@@ -40,19 +38,6 @@ internal val contextDependentModule = module {
         KeyPairGeneratorSpec.Builder(androidContext())
     }
 
-    single<OfflineSyncer>(createOnStart = true) {
```

Comment:
Why are you removing this? We need it to have prefilled some DB tables and to have sync of some remote data. 🤔

Reply:
I brought it back. Good point.

Reply:
It´s back